rental.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'book'
+require_relative 'person'
+
+# Rental represents a book rental by a person on a specific date.
+class Rental
+  attr_accessor :date, :book, :person
+
+  # Initializes a new Rental object.
+  # date: The date of the rental.
+  # book: The book being rented.
+  # person: The person renting the book.
```

Comment:
- [ ] Kindly consider removing all the comments from your code so it's easy to navigate and looks clean